provisionerd/runner/runner.go

```diff
@@ -1116,6 +1116,7 @@ func (r *Runner) failedJobf(format string, args ...interface{}) *proto.FailedJob
 		JobId:     r.job.JobId,
 		Error:     message,
 		ErrorCode: code,
+		Type:      &proto.FailedJob_WorkspaceBuild_{},
```

Comment:
I was able to trigger this by using the Docker template, creating a workspace, deleting my Docker volumes, then trying to start the workspace again.

It does not trigger through the unit test.  Not sure how to make it take this code path.